newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +134,38 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
```

Comment:
See also my other comments about handling this boolean, you'll need to replace `Promise<void>` by `Promise<boolean>`